server/src/utils/jobReminderCron.js

```diff
@@ -17,33 +20,58 @@ let jobCron = new CronJob({
 });
 
 let messageUsers = () => {
-  User.findAll()
-  .then(users =>{
+  User.findAll({
+    include: [{
+      model: Tag,
+      include: [{
+        model: Job,
+        where: {
+          createdAt: {
+            $gt: new Date(new Date() - 24 * 60 * 60 * 1000)
+          }
+        }
+      }]
+    }]
+  })
+  .then(users => {
     users.forEach(user => {
+      let count ={};
+      const tagArr = user.dataValues.tags.map(item => {
+        return item.name
+      });
       const id = user.dataValues.slackUserId;
       const BUCKLEY = store[user.dataValues.slackTeamId];
 
-      Tag.findOne({
-        // TODO: Figure out how to make this a join table
-        where: { name: 'javascript' },
-        include: [{ model: Job }],
+      user.dataValues.tags.forEach(tag => {
+        tag.jobs.forEach(job => {
+          if (count[job.id]) { count[job.id]++; }
```

Comment:
please fix this structure -- one liner if statements with else blocks hurt my brain